--- a/install.sh
+++ b/install.sh
@@ -5,11 +5,11 @@ if [[ $EUID -ne 0 ]]; then
    exit 1
 fi
 
-is_Raspberry=$(cat /proc/device-tree/model | awk  '{print $1}')
-if [ "x${is_Raspberry}" != "xRaspberry" ] ; then
-  echo "Sorry, this drivers only works on raspberry pi"
-  exit 1
-fi
+#is_Raspberry=$(cat /proc/device-tree/model | awk  '{print $1}')
+#if [ "x${is_Raspberry}" != "xRaspberry" ] ; then
+#  echo "Sorry, this drivers only works on raspberry pi"
+#  exit 1
+#fi
 
 ver="1.0"
 
